roles/download/defaults/main.yml

```diff
@@ -50,7 +50,7 @@ image_arch: "{{host_architecture | default('amd64')}}"
 
 # Versions
 kubeadm_version: "{{ kube_version }}"
-etcd_version: v3.4.13
```

Comment:
ETCD should not be updated as k8s 1.21 still use 3.4.13, please wait for 1.22 ;)

Reply:
So this would mean that we need to modify the way we calculate `etcd_version` based on `kube_version` when 1.22 comes out and we add support.

Reply:
We could indeed add some kind of "if"

Reply:
I have it with 1.21.2 and it works well. Is some requirement site for this?